### PR TITLE
Cross validation in Elastic Net

### PR DESCRIPTION
**Pull request by iyerr3**
Note: Currently this is rebased on top of the grouping PR (#69) and has the commits from that PR. When that PR is merged, this should be corrected to only the CV commits.

### CODE REVIEW

**Comment by iyerr3** [comment]
@njayaram2 
Apologies for the major history change. Rebasing to master required squashing commits (to ease the rebase). Your comment related to independent variables naming should be addressed now. 
Please continue with your review.

File: src/ports/postgres/modules/utilities/in_mem_group_control.py_in [hunk 1]
```diff
@@ -333,9 +335,15 @@ class GroupIterationController:
...
     def are_last_state_value_zero(self):
         return self.new_states.are_last_state_value_zero()
-    
+
+=======
+    def get_last_state_value(self):
+        return self.new_states.get_last_state_value()
+
+>>>>>>> 63616e3... Elastic net: Add cross validation
     def info(self):
```
Comment (njayaram2) [inline src/ports/postgres/modules/utilities/in_mem_group_control.py_in]: Is there something wrong with the rebase here?

Comment (iyerr3) [inline src/ports/postgres/modules/utilities/in_mem_group_control.py_in]: Thanks. Fixed now.

File: src/ports/postgres/modules/validation/internal/cross_validation.py_in [hunk 2]
```diff
@@ -336,22 +339,31 @@ class CrossValidator(object):
...
 
-        predictor(schema_madlib, model_table,
-                  rel_valid, col_id, output_table)
+        model_vals = plpy.execute("SELECT * FROM " + model_table)
+        plpy.info("---- Model table {0} = {1}".format(model_table, str(model_vals)))
+        data_cols = get_cols_and_types(rel_valid)
+        plpy.info("---- Source table columns = " + str(data_cols))
+
+        plpy.info("--------------------------------------------")
+        predictor(schema_madlib, model_table, rel_valid, col_id, output_table)
```
Comment (njayaram2) [inline src/ports/postgres/modules/validation/internal/cross_validation.py_in]: The various `plpy.info()` calls are flooding the interface with lots of messages. Do we want to print out these info strings all the time?

Comment (iyerr3) [inline src/ports/postgres/modules/validation/internal/cross_validation.py_in]: Nope - those are unnecessary debug statements.

**Comment by njayaram2** [comment]
+1 LGTM

**Pull request closed without merging**